recipes/header/settings_menu_button/settings_menu_button.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <dt-button
+    v-if="updateAvailable"
+    importance="outlined"
+    size="xs"
+    icon-position="right"
+    class="d-h32 d-bar-pill d-bgc-blue-200 d-bgo25 h:d-bgo50 d-bc-blue-500 d-bco25 d-fc-blue-500"
+  >
+    <slot />
+    <template #icon>
+      <dt-icon
+        name="more-vertical"
+        size="400"
+        class="d-bs-none d-fc-blue-500 d-ol-none d-svg--size20"
+      />
+    </template>
+  </dt-button>
+  <dt-button
+    v-else
+    importance="clear"
+    kind="inverted"
+    class="settings-menu-button d-p0 d-bar24 d-h32 d-w24 d-theme-topbar-fc"
+  >
+    <template #icon>
+      <dt-icon
+        name="more-vertical"
+        size="400"
+        class="d-bs-none d-ol-none d-svg--size20"
+      />
+    </template>
+  </dt-button>
+</template>
+
+<script>
+import DtButton from '@/components/button/button';
+import { DtIcon } from '@/components/icon';
+
+export default {
+  name: 'SettingsMenuButton',
+
+  components: {
+    DtButton,
+    DtIcon,
+  },
+
+  props: {
+    /**
+     * Determines whether the button should display the update state
+     * default is false
+     * @values true, false
+     */
+    updateAvailable: {
+      type: Boolean,
+      default: false,
+    },
+  },
+
+  emits: [
+    /**
+     * Native click event
+     *
+     * @event click
+     * @type {PointerEvent | KeyboardEvent}
+     */
+    'click',
+  ],
+
+};
+</script>
+
+<style scoped lang="less">
+.d-btn:active {
```

Comment:
This doesn't feel like the right way to apply styling during a click, but I couldn't get the active prop to work.

Reply:
Yes, that's per design. None of the header buttons in that style have active styling